src/PixiEditor/Models/Dialogs/ExportFileDialog.cs

```diff
@@ -78,11 +78,19 @@ public override bool ShowDialog()
     {
         ExportFilePopup popup = new ExportFilePopup(FileWidth, FileHeight);
         popup.ShowDialog();
+
         if (popup.DialogResult == true)
         {
             FileWidth = popup.SaveWidth;
             FileHeight = popup.SaveHeight;
             FilePath = popup.SavePath;
+
+            if (ChosenFormat.ToString() != "pixi")
+            {
+                ExportWarningPopup exportWarningPopup = new ExportWarningPopup();
```

Comment:
This is not a good solution. PixiEditor already has dialog popups that you can customize.